face_recognition/api.py

```diff
@@ -77,6 +79,20 @@ def load_image_file(filename, mode='RGB'):
     """
     return scipy.misc.imread(filename, mode=mode)
 
+def load_image_url(url, mode='RGB'):
+    """
+    Loads an image data from url into a numpy array 
+
+    :param url: image url to load
+    :param mode: format to convert the image to. Only 'RGB' (8-bit RGB, 3 channels) and 'L' (black and white) are supported.
+    :return: image contents as numpy array
+    """
+    opener = build_opener()
+    opener.addheaders = [('User-Agent', 'Mozilla/5.0')]
```

Comment:
You shouldn't lie in the User-Agent, this should by something like `python/face_recognition/<version>`